desmume/src/frontend/posix/gtk/graphics.ui

```diff
@@ -94,14 +94,49 @@
                 <property name="top_attach">1</property>
               </packing>
             </child>
+            <child>
+              <object class="GtkLabel">
+                <property name="label" translatable="yes">GPU scale factor:</property>
+              </object>
+              <packing>
+                <property name="left_attach">0</property>
+                <property name="top_attach">2</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkComboBoxText" id="gpuscale">
+                <items>
+                  <item translatable="yes">×1</item>
```

Comment:
wouldn't it be better to make the scale factor a float, like in cli, and use some sort of editbox with a knob for the selection, instead of hardcoding a few ints in a combobox ?